base/src/main/java/com/smartdevicelink/managers/screen/choiceset/ChoiceSet.java

```diff
@@ -398,4 +399,12 @@ private void checkChoiceSetParameters(){
         return clonedHelpItems;
     }
 
+    private Vector<TTSChunk> createSimpleTTSChunks(String simple) {
+        if (simple == null) {
+            return null;
+        }
+        Vector<TTSChunk> chunks = new Vector<>();
+        chunks.add(new TTSChunk(simple, SpeechCapabilities.TEXT));
+        return chunks;
+    }
```

Comment:
Can We remove this method and replace its called with the proposed one liner 
`Collections.singletonList(new TTSChunk(simple, SpeechCapabilities.TEXT));`